deploy/kubernetes/elasticsearch/kubernetes-elasticsearch-all-in-one.yaml

```diff
@@ -0,0 +1,3011 @@
+---
+# Downloaded from https://download.elastic.co/downloads/eck/1.2.0/all-in-one.yaml
+# From the instructions in https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-deploy-eck.html
+# Do not modify
```

Comment:
I copied this file directly from the link in the comment here. In theory, we could just use that URL in our deployment script, but in the past we have had issues with deployment stability when we rely on external resources, so I put a copy in our source control so we will always have a stable version to rely on